README.md

```diff
@@ -1,75 +1,53 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# To do list App
 
-# Project Name
-
-> One paragraph statement about the project.
+> Website to storage your favorite books.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML-CSS-JAVASCRIPT-WEBPACK
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Visit Live Demo Link](https://felipeg005.github.io/To-Do-List-ES6/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+To get a local copy up and running follow these simple example steps
 
 ### Prerequisites
 
-### Setup
-
-### Install
```

Comment:
- Would be helpful to have these points in your README.

Reply:
Thanks, you're totally right I will fix it.